tutorials/Bert_SQUAD_Interpret.ipynb

```diff
@@ -0,0 +1,1111 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's interpret parent `BertEmbeddings` layer. For that reason we separate embedding layer and precompute the embeddings. `configure_interpretable_embedding_layer` function helps us to do that. It returns `InterpretableEmbeddingBase` layer that wraps `BertEmbeddings` and can be used to access the embedding vectors. Note that we need to remove `InterpretableEmbeddingBase` wrapper from our model using `remove_interpretable_embedding_layer` function after we finish interpretation.\n",
+    "\n",
+    "Note that there is a way to interpret BertEmbeddings without configuring interpretable embedding layers that we will make available soon, However it is important to not that configuring interpretable embedding layers gives us more flexibility."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/narine/captum/captum/attr/_models/base.py:181: UserWarning: In order to make embedding layers more interpretable they will\n",
+      "        be replaced with an interpretable embedding layer which wraps the\n",
+      "        original embedding layer and takes word embedding vectors as inputs of\n",
+      "        the forward function. This allows to generate baselines for word\n",
+      "        embeddings and compute attributions for each embedding dimension.\n",
+      "        The original embedding layer must be set\n",
+      "        back by calling `remove_interpretable_embedding_layer` function\n",
+      "        after model interpretation is finished.\n",
+      "  after model interpretation is finished.\"\"\"\n"
+     ]
+    }
+   ],
+   "source": [
+    "interpretable_embedding = configure_interpretable_embedding_layer(model, 'bert.embeddings')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Defining and numericalizing / encoding special tokens"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ref = \"[PAD]\" # A token used for generating token reference\n",
+    "sep = \"[SEP]\" # A token used as a separator between question and text and it is also added to the end of the text.\n",
+    "cls = \"[CLS]\" # This token is prepended to the concatenated question-text word sequence\n",
+    "\n",
+    "ref_token_id = tokenizer.encode(ref)\n",
+    "sep_token_id = tokenizer.encode(sep)\n",
+    "cls_token_id = tokenizer.encode(cls)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below we define a set of helper function fo constructing refenreces / baselines for word tokens, token types and position ids. We also provide separate helper functions that allow to construct the sub-embeddings and corresponding refenrences for all subembeddings of `BertEmbeddings` layer."
```

Comment:
Nit: `fo constructing refenreces` -> `for constructing references`

Reply:
Nit: `corresponding refenrences for all subembeddings` -> `corresponding references for all sub-embeddings`